src/website/app/demos/Tooltip/bestPractices.js

```diff
@@ -0,0 +1,186 @@
+dislikes, or avoids pleasure itself, because it is pleasure, but because
+those who do not know how to pursue pleasure rationally encounter
+consequences that are extremely painful. Nor again is there anyone who
+loves or pursues or desires to obtain pain of itself, because it is pain,
+but because occasionally circumstances occur in which toil and pain can
+procure him some great pleasure. To take a trivial example, which of us
+ever undertakes laborious physical exercise, except to obtain some
+advantage from it? But who has any right to find fault with a man who
+chooses to enjoy a pleasure that has no annoying consequences, or one who
+avoids a pain that produces no resultant pleasure?”
+`;
+
+const tooltipEveryFifthWord = words => {
+  function addSpaces(word, index) {
+    return (1 === index ? '' : ' ') + `${word} `;
+  }
+
+  return words.split(/[ \n]/).reduce((acc, word, index) => {
+    const possiblyTooltipedWord =
+      0 === index % 5 ? (
+        <Tooltip key={index} content={word} placement="top">
+          {word}
+        </Tooltip>
+      ) : (
+        addSpaces(word, index)
+      );
+    return acc.concat(possiblyTooltipedWord);
+  }, []);
+};
+
+export default [
+  {
+    type: 'do',
+    description:
+      'Use Tooltips to provide extra information, such as titles and keyboard shortcuts, for selected interface elements.',
+    example: (
+      <div>
+        <Tooltip content="Bold (⌘B)">
+          <Button minimal iconStart={<IconFormatBold title="Bold" />} />
+        </Tooltip>
+        <Tooltip content="Italic (⌘I)">
+          <Button minimal iconStart={<IconFormatItalic title="Italic" />} />
+        </Tooltip>
+        <Tooltip content="Underlined (⌘U)">
+          <Button
+            minimal
+            iconStart={<IconFormatUnderlined title="Underlined" />}
+          />
+        </Tooltip>
+        <Tooltip content="Left align (⌘+Shift+L)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignLeft title="Left align" />}
+          />
+        </Tooltip>
+        <Tooltip content="Center align (⌘+Shift+E)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignCenter title="Center align" />}
+          />
+        </Tooltip>
+        <Tooltip content="Right align (⌘+Shift+R)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignRight title="Right align" />}
+          />
+        </Tooltip>
+        <Tooltip content="Justify (⌘+Shift+J)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignJustify title="Justify" />}
+          />
+        </Tooltip>
+      </div>
+    )
+  },
+  {
+    type: 'dont',
+    description: `Don't use Tooltips to hide captions, subtitles, statuses, or secondaryText.
+Information hidden behind a hover interaction is much less likely to be discovered by the user.
+Important information should be displayed plainly on the page.
```

Comment:
Suggest to reword first sentence - "Don't use Tooltips to hide important information."
Suggest to delete last sentence.

Reply:
Done.